### PR TITLE
Remove metrics that are no longer required

### PR DESCRIPTION
**Pull request by rudigiesler**
There is a race condition in the caching of metrics that are expensive to calculate. These metrics are no longer required, so we can just remove them instead of fixing the issue

### CODE REVIEW

File: subscriptions/tests.py [hunk 1]
```diff
@@ -2013,46 +2005,31 @@ def test_metrics_read(self):
         # Check
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+        self.maxDiff = None
+        for r in sorted(response.data['metrics_available']):
+            print(r)
```
Comment (erikh360) [inline subscriptions/tests.py]: You can probably remove this print and I can't see where `self.maxDiff` is used?

File: seed_stage_based_messaging/settings.py [hunk 2]
```diff
@@ -234,37 +219,26 @@
 
 METRICS_REALTIME = [
     'subscriptions.created.sum',
-    'subscriptions.send_next_message_errored.sum'
+    'subscriptions.send_next_message_errored.sum',
+    'sbm.send_next_message.connection_error.sum',
```
Comment (erikh360) [inline seed_stage_based_messaging/settings.py:223]: Not sure I understand why these were renamed?

**Review by erikh360 — COMMENTED** [review]
Just the two comments but looks good.

File: subscriptions/tests.py (hunk 1 above)
Comment (rudigiesler) [reply subscriptions/tests.py]: Whoops, that was just me debugging the tests, forgot to remove. `maxDiff` allows you to see the full diff from `assertEqual`

File: seed_stage_based_messaging/settings.py (hunk 2 above)
Comment (rudigiesler) [reply seed_stage_based_messaging/settings.py:223]: They weren't renamed, they were just wrong in the first place. The task fires them as `sbm...`, but in the metrics list they appeared as `subscriptions...`, so if we tried to use these metrics, they wouldn't have worked.

**Review by erikh360 — APPROVED** [review]
🍰 when travis agrees.

**Pull request merged**